source/localizable/routing/redirection.md

```diff
@@ -1,24 +1,34 @@
-Calling [`transitionTo()`][1] from a route or [`transitionToRoute()`][2] from a 
-controller will stop any transition currently in progress and start a new 
-one, functioning as a redirect. `transitionTo()` behaves exactly like the 
-[link-to](../../templates/links) helper.
+Sometimes you want to redirect a user to a different page than what they requested for.
 
-[1]: http://emberjs.com/api/classes/Ember.Route.html#method_transitionTo
-[2]: http://emberjs.com/api/classes/Ember.Controller.html#method_transitionToRoute
+For example, if they're not logged in, you might want to prevent them from editing their profile, accessing private information,
+or checking out items in their shopping cart.
+Usually you want to redirect them to the login page, and after they have successfully logged in, take them back to the page they originally wanted to access.
+Another use case could be preventing a user who has already logged in from visiting the login page again.
+
+There's many other reasons you probably want to have the last word on whether a user can or cannot access a certain page.
```

Comment:
I think this sentence and the following could the condensed into something like "There are many reasons you might want to control user access to a certain page, and Ember provides APIs to do so in your route."